src/diff/index.js

```diff
@@ -385,17 +384,17 @@ function diffElementNodes(
 		if (!isHydrating) {
 			if (
 				'value' in newProps &&
-				newProps.value !== undefined &&
-				newProps.value !== dom.value
+				(i = newProps.value) !== undefined &&
+				i !== dom.value
 			) {
-				dom.value = newProps.value == null ? '' : newProps.value;
+				setProperty(dom, 'value', i == null ? '' : i, oldProps.value, false);
```

Comment:
```suggestion
				setProperty(dom, 'value', i, oldProps.value, false);
```

We do this in `setProperty`

Reply:
Oh true! Good call - thanks :)